### PR TITLE
docs - misc improvements and clarifications

### PR DESCRIPTION
**Pull request by lisakowen**
resolves:
https://github.com/greenplum-db/sre-test/issues/251
https://github.com/greenplum-db/sre-test/issues/252
https://github.com/greenplum-db/sre-test/issues/255
https://github.com/greenplum-db/sre-test/issues/257
https://github.com/greenplum-db/sre-test/issues/273
https://github.com/greenplum-db/sre-test/issues/281
https://github.com/greenplum-db/sre-test/issues/255

### CODE REVIEW

File: docs/content/cfghostport.html.md.erb [hunk 1]
```diff
@@ -44,3 +44,5 @@ Perform the following procedure to configure the PXF Service host and/or port nu
 
 4. Restart PXF on each Greenplum Database host as described in [Restarting PXF](cfginitstart_pxf.html#restart_pxf).
 
+5. Verify that PXF is running on the reconfigured host and/or port by invoking `http://<PXF_HOST>:<PXF_PORT>/<endpoint>[/<name>]` to view PXF monitoring information as described in [About PXF Service Runtime Monitoring](monitor_pxf.html#about_rtm).
```
Comment (denalex) [inline docs/content/cfghostport.html.md.erb:47]: `<endpoint>[/<name>]` this is not user-friendly, they will not know what to put here, maybe we can provide the specific endpoint values here

File: docs/content/hdfs_orc.html.md.erb [hunk 2]
```diff
@@ -133,12 +133,12 @@ Procedure:
...
     ```
 
-1. [Download](https://repo1.maven.org/maven2/org/apache/orc/orc-tools/1.6.2/orc-tools-1.6.2-uber.jar) the `orc-tools` JAR.
+1. [Download](https://repo1.maven.org/maven2/org/apache/orc/orc-tools) the most recent version of the `orc-tools` JAR to the current working directory.
 
 1. Run the `orc-tools` `convert` command to convert `sampledata.json` to the ORC file `/tmp/sampledata.orc`; provide the schema to the command:
 
     ``` shell
-    hdfsclient$ java -jar orc-tools-1.6.2-uber.jar convert /tmp/sampledata.json \
+    hdfsclient$ java -jar orc-tools-1.7.3-uber.jar convert /tmp/sampledata.json \
```
Comment (denalex) [inline docs/content/hdfs_orc.html.md.erb:141]:
```suggestion
    $ java -jar orc-tools-1.7.3-uber.jar convert /tmp/sampledata.json \
```

**Review by denalex — APPROVED** [review]

File: docs/content/pxfuserimpers.html.md.erb [hunk 3]
```diff
@@ -115,6 +115,8 @@ Perform the following procedure to configure the Hadoop user:
     </property>
     ```
 
+    The Hadoop user `hdfsuser1` must exist in the Hadoop cluster.
```
Comment (ashuka24) [inline docs/content/pxfuserimpers.html.md.erb:118]: Is it safe to assume that one will know how to create the Hadoop user?

File: docs/content/hdfs_avro.html.md.erb [hunk 4]
```diff
@@ -262,13 +262,17 @@ Perform the following steps to create a sample Avro data file conforming to the
...
+    1. Download the most recent version of the Avro tools jar from http://avro.apache.org/releases.html to the current working directory.
+
+    1. Convert the file:
+
+        ``` shell
+        $ java -jar ./avro-tools-1.11.0.jar fromjson --schema-file /tmp/avro_schema.avsc /tmp/pxf_avro.txt > /tmp/pxf_avro.avro
+        ```
 
-    The generated Avro binary data file is written to `/tmp/pxf_avro.avro`. 
+        The generated Avro binary data file is written to `/tmp/pxf_avro.avro`.
```
Comment (ashuka24) [inline docs/content/hdfs_avro.html.md.erb:275]: why is this extra indent needed?

**Review by ashuka24 — APPROVED** [review]

File: docs/content/hdfs_avro.html.md.erb (hunk 4 above)
Comment (lisakowen) [reply docs/content/hdfs_avro.html.md.erb:275]: the indent is needed to align this text with the subcommand.

File: docs/content/pxfuserimpers.html.md.erb (hunk 3 above)
Comment (lisakowen) [reply docs/content/pxfuserimpers.html.md.erb:118]: the user configuring impersonation might not be the user who admins the hadoop cluster.  this statement is just stating a requirement that the user who is configuring pxf should verify.

**Comment by lisakowen** [comment]
thank you for reviewing, @ashuka24 and @denalex

File: docs/content/hdfs_avro.html.md.erb [hunk 5]
```diff
@@ -262,13 +262,17 @@ Perform the following steps to create a sample Avro data file conforming to the
...
 
-3. Convert the text file to Avro format. There are various ways to perform the conversion, both programmatically and via the command line. In this example, we use the [Java Avro tools](http://avro.apache.org/releases.html); the jar `avro-tools-1.9.1.jar` file resides in the current directory:
+3. Convert the text file to Avro format. There are various ways to perform the conversion, both programmatically and via the command line. In this example, we use the Java Avro tools.
 
-    ``` shell
-    $ java -jar ./avro-tools-1.9.1.jar fromjson --schema-file /tmp/avro_schema.avsc /tmp/pxf_avro.txt > /tmp/pxf_avro.avro
-    ```
+    1. Download the most recent version of the Avro tools jar from http://avro.apache.org/releases.html to the current working directory.
+
+    1. Convert the file:
```
Comment (hpandeycodeit) [inline docs/content/hdfs_avro.html.md.erb:269]: nit: 
```suggestion
    2. Convert the file:
```

Comment (lisakowen) [reply docs/content/hdfs_avro.html.md.erb:269]: hey @hpandeycodeit - our markdown tool auto-numbers.  i have been trying to remember to use 1 for new steps that i add.  maybe i should just change to 1. for all steps in files incrementally as i update them.

**Review by hpandeycodeit — APPROVED** [review]

**Pull request merged**